config/routes.rb

```diff
@@ -3,4 +3,6 @@
   post "/items", to: "items#index"
 
   resource :cart
+  get '/login', to: 'sessions#new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.